quizzes/07_parameters.test.js

```diff
@@ -0,0 +1,50 @@
+console.log(
+  // randGen();
+  // weatherOnDemand(arr, 'Bali', 'London')``
```

Comment:
Does this work? `arr` should probably be defined before, or turn it into a function declaration that we can call to get `arr`. Also, I think that this line has a typo (and the line above). I want to be able to uncomment the line really quickly :)